apps/api/src/app/agents/services/telegram-mobile-link-token.service.ts

```
@@ -0,0 +1,155 @@
+import { randomUUID } from 'node:crypto';
+import { Injectable, Logger } from '@nestjs/common';
```

Biome (2.4.15)

[error] Please use the PinoLogger from `@novu/application-generic` instead

(lint/style/noRestrictedImports)